frontend/src/metabase-lib/lib/queries/StructuredQuery.js

```diff
@@ -306,6 +306,7 @@ export default class StructuredQuery extends AtomicQuery {
           column =>
             new Field({
               ...column,
+              // TODO FIXME -- Do NOT use field-literal unless you're referring to a native query
```

Comment:
Is this something that needs to be addressed NOW (before the merge)?

Reply:
No, we can work around it until it gets fixed. I tried fixing it but it broke other stuff in the front end

Reply:
https://github.com/metabase/metabase/issues/14824